testonly/nodes.go

```diff
@@ -0,0 +1,13 @@
+package testonly
+
+import (
+	"github.com/google/trillian/storage"
+)
+
+func MustCreateNodeIDForTreeCoords(depth, index int64, maxBitLen int) storage.NodeID {
```

Comment:
Small consistency thing - in definition of NewNodeIDForTreeCoords the third param is called 'maxTreeDepth', maybe use that here too?